src/interface/src/app/treatments/direct-impacts/direct-impacts.component.ts

```diff
@@ -121,7 +121,21 @@ export class DirectImpactsComponent implements OnInit, OnDestroy {
     })
   );
 
-  activateMetric(data: MapMetric) {
+  activeMetric$ = this.directImpactsStateService.activeMetric$.pipe(
+    map((m) => m.metric)
+  );
+
+  // todo: placeholder to fill once we have project area filter
+  projectArea$ = of('All Project Areas');
```

Comment:
:+1: I also have a placeholder brewing in a different format, but we can definitely sort that out later.